das/aggregator.go

```diff
@@ -103,10 +104,11 @@ func NewAggregatorWithL1Info(
 	if err != nil {
 		return nil, err
 	}
-	return NewAggregatorWithSeqInboxCaller(config, services, seqInboxCaller)
+	return NewAggregatorWithSeqInboxCaller(ctx, config, services, seqInboxCaller)
 }
 
 func NewAggregatorWithSeqInboxCaller(
+	ctx context.Context,
```

Comment:
Decided to remove this context, so people don't mistakenly think that canceling it will have some effect.  This caused cascading removals of context from some other APIs, for the same reason.